Snowflake.Data.Tests/UnitTests/SnowflakeDbConnectionTest.cs

```diff
@@ -0,0 +1,61 @@
+
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    using Core;
+    using Core.Tools;
+    using Moq;
+    using NUnit.Framework;
+    using Snowflake.Data.Client;
+
+    public class SnowflakeDbConnectionTest
+    {
+        [Test]
+        public void TestFillConnectionStringFromTomlConfig()
```

Comment:
maybe you could rename it to express better what use case is this test covering. Maybe TestTomlConfigurationNotAplliedForConnectionWithConnectionString ?
But actually the next test is covering the same use case. So there is missing a case when configuration from toml is applied.